ui/app/components/tool-lookup.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class HashTool extends Component {
```

Comment:
Sort of a nit, but can we name the class to match the file and how they're invoked (eg this one would be `ToolLookup`)

Reply:
I totally missed that, not a nit at all.  Nice catch, thanks!